src/shared/cgroup-setup.c

```diff
@@ -489,3 +489,16 @@ int cg_has_legacy(void) {
                                "Unknown filesystem type %llx mounted on /sys/fs/cgroup/.",
                                (unsigned long long) fs.f_type);
 }
+
+int cg_is_ready(void) {
```

Comment:
doesn't matter, but maybe call this `cg_is_available()`? I mean, cgroups might be ready but not available because we are running in a chroot without it mounted. this checks if the thing is available locally, hence I think "available" is a better name